src/diffusers/pipelines/wan/pipeline_wan_i2v.py

```diff
@@ -599,7 +619,7 @@ def __call__(
         if negative_prompt_embeds is not None:
             negative_prompt_embeds = negative_prompt_embeds.to(transformer_dtype)
 
-        image_embeds = self.encode_image(image, device)
+        image_embeds = self.encode_image(image, image_embeds, device)
```

Comment:
```suggestion
        if not image_embeds:
            image_embeds = self.encode_image(image, image_embeds, device)
```

can we do this? (and not adding a new argument to `encode_image`)